Modules/TOF/src/TOFDecoderCompressed.cxx

```diff
@@ -0,0 +1,87 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   TOFDecoderCompressed.cxx
+/// \author Nicolo' Jacazio
+///
+
+// O2 includes
+#include "DataFormatsTOF/CompressedDataFormat.h"
+#include <Framework/DataRefUtils.h>
+#include "Headers/RAWDataHeader.h"
+#include "DetectorsRaw/HBFUtils.h"
+
+using namespace o2::framework;
+
+// QC includes
+#include "QualityControl/QcInfoLogger.h"
+#include "TOF/TOFDecoderCompressed.h"
+
+// #define VERBOSEDECODERCOMPRESSED //Flag for a verbose Decoder
+
+namespace o2::quality_control_modules::tof
+{
+
+void TOFDecoderCompressed::decode()
+{
+  DecoderBase::run();
+}
+
+void TOFDecoderCompressed::headerHandler(const CrateHeader_t* crateHeader, const CrateOrbit_t* /*crateOrbit*/)
+{
+  for (int ibit = 0; ibit < 11; ++ibit)
```

Comment:
We advise to use accolades in statements `for` and `if` independently from the number of lines.

Reply:
Done!